builtins/amp-img.js

```diff
@@ -123,9 +123,10 @@ export class AmpImg extends BaseElement {
     // only read "Graphic" when using only 'alt'.
     if (this.element.getAttribute('role') == 'img') {
       this.element.removeAttribute('role');
-      user().error('AMP-IMG', 'Setting role=img on amp-img elements breaks ' +
-        'screen readers please just set alt or ARIA attributes, they will ' +
-        'be correctly propagated for the underlying <img> element.');
+      user(this.element).error('AMP-IMG',
```

Comment:
Question: Do we plan to migrate the new `user(opt_element)` in this PR?  If so why we only make the change to `amp-img.js`. I would suggest migrate to new format in separate PR.